src/Metalama/Metalama.Compiler.Sdk/Metalama.Compiler.Sdk.csproj

```diff
@@ -10,6 +10,7 @@
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <!-- Enable portable PDB so it can be included in the symbols NuGet package (.snupkg). -->
     <DebugType>Portable</DebugType>
+    <NoWarn>$(NoWarn);NU5100</NoWarn>
```

Comment:
Comment on what these warnings mean, why they are suppressed, and when (if ever) this suppression should be removed.